x/wasm/keeper/test_common.go

```diff
@@ -349,18 +363,17 @@ func createTestInput(
 		scopedIBCKeeper,
 	)
 
-	router := baseapp.NewRouter()
-	bh := bank.NewHandler(bankKeeper)
-	router.AddRoute(sdk.NewRoute(banktypes.RouterKey, bh))
-	sh := staking.NewHandler(stakingKeeper)
-	router.AddRoute(sdk.NewRoute(stakingtypes.RouterKey, sh))
-	dh := distribution.NewHandler(distKeeper)
-	router.AddRoute(sdk.NewRoute(distributiontypes.RouterKey, dh))
+	router := authmiddleware.NewLegacyRouter()
+	br := bank.AppModule.Route(bank.AppModule{})
+	router.AddRoute(br)
+	sr := staking.AppModule.Route(staking.AppModule{})
+	router.AddRoute(sr)
+	dr := distribution.AppModule.Route(distribution.AppModule{})
+	router.AddRoute(dr)
```

Comment:
sorry, I meant all of this.  It is throwing deprecated errors from golangci-lint